packages/core/lib/commands/install/meta.js

```diff
@@ -1,9 +1,10 @@
 module.exports = {
   command: "install",
-  description: "Install a package from the Ethereum Package Registry",
+  description:
+    "[deprecated] Install a package from the Ethereum Package Registry",
   builder: {},
   help: {
-    usage: "truffle install <package_name>[@<version>]",
+    usage: "[deprecated] truffle install <package_name>[@<version>]",
```

Comment:
It seems odd to describe how to use a command that a user can't because it isn't implemented. Does a deprecated command need `usage/description/options`? What do y'all think?

```console
$ truffle help install

  Usage:        [deprecated] truffle install <package_name>[@<version>]
  Description:  [deprecated] Install a package from the Ethereum Package Registry
  Options:
                package_name
                    Name of the package as listed in the Ethereum Package Registry. (required)
                <@version>
                    When specified, will install a specific version of the package, otherwise will install
                    the latest version.
```

Reply:
I guess @cds-amal, we can just remove `help` and only keep the command `description` and mention as `deprecated`.

Reply:
@eggplantzzz thoughts?

Reply:
I'd be curious what other libraries do, but if the command no longer exists, I'd personally say it's reasonable to remove the `help` as well.

Reply:
> I guess @cds-amal, we can just remove `help` and only keep the command `description` and mention as `deprecated`.

Where would the `description` be used if `help` is removed? Maybe it's better to not display help for these commands and indicate deprecation in the stubbed implementation.

Reply:
My first thought is that @cds-amal's suggestion of removing it from the help and providing messaging in the stubbed implementation is good. We probably shouldn't advertise it if it is broken/deprecated.